apps/fft/fft_generator.cpp

```diff
@@ -142,11 +143,13 @@ class FFTGenerator : public Halide::Generator<FFTGenerator> {
         const int input_comps = (input_number_type == FFTNumberType::Real) ? 1 : 2;
         const int output_comps = (output_number_type == FFTNumberType::Real) ? 1 : 2;
 
+        // clang-format off
         input.dim(0).set_stride(input_comps)
```

Comment:
Annoying that all these dim() chains require turning off clang-format. How do you feel about doing the following as an alternative idiom:

```
input.dim(0).set_foo(...).set_bar(...);
input.dim(2).set_foo(...).set_bar(...);
```